src/SB/Game/zNPCTypeBossPlankton.cpp

```diff
@@ -988,6 +1072,33 @@ void zNPCBPlankton::Init(xEntAsset* asset) //66%
     zNPCBPlankton::aim_gun(play, &gun_tilt, &move.dest, 0);
 }
 
+void zNPCBPlankton::Setup()
+{
+    U32 tmpVar;
+
+    zNPCBoss::Setup();
+    zNPCBPlankton::setup_beam();
+    tmpVar = xStrHash("NPC_NEWSCASTER");
```

Comment:
It looks like the regression is because this string comes before all the debug tweak strings the rodata and now the diff algorithm doesn't see the tweak string data as matching. I'm not gonna worry about it for now since this file is clearly very WIP but it might be as simple as a function ordering issue.